test/test.js

```diff
@@ -1,7 +1,46 @@
 var Logsene = require('../index.js')
 var token = process.env.LOGSENE_TOKEN
```

Comment:
The tests pass in `master` branch without a token set because master allows the Logsene object to be constructed without a token, which seems to be ok as far as the test is concerned. 

If keeping this behavior is ok, I suggest changing this line to be:

    var token = process.env.LOGSENE_TOKEN || 'test-token';

This should work the same.